delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/Resolvers.scala

```diff
@@ -111,6 +114,15 @@ trait Resolvers {
       case None           => IO.pure(None)
     }
 
+  /**
+    * Lists all resolvers.
+    *
+    * @param pagination the pagination settings
+    * @param params     filter parameters for the listing
+    * @return a paginated results list
+    */
+  def list(pagination: FromPagination, params: ResolverSearchParams): UIO[UnscoredSearchResults[ResolverResource]]
```

Comment:
At the API level, `Resolvers` are accessed in the scope of a project. So we probably need 2 things:
1. `ResolverSearchParams` should have a parameter which is `projectRef: ProjectRef`
2. 
```scala
def list(projectRef: ProjectRef, pagination: FromPagination, params: ResolverSearchParams): UIO[UnscoredSearchResults[ResolverResource]] - 
   list(pagination, params.copy( projectRef= Some(projectRef)))
```